e2e-tests/package.json

```diff
@@ -6,10 +6,10 @@
   "scripts": {
     "clean": "rm -rf build node_modules",
     "build": "npm i",
-    "test:create": "tsx jasmine spec/create-survey.spec.ts",
```

Comment:
I'm a bit confused by `build` only `installing` and pretest` only compiling, and as to why we need `tsx` if we're calling `tsc` to compile. 

Here's how I'd expect things to work, please cmiiw;

```
"prebuild": "npm install",
"build": "npm run compile",
"pretest: "npm run build",
"test:create-survey": "jasmine dist/spec/create-survey.spec.ts"
"test:create-survey:debug": "npm run test:create-survey -- --inspect" <-- not sure this works
```

Wdyt?

Reply:
Ah, sorry for the random boilerplate...`tsx`  lets you run `.ts` files directly in node without any compile step in between, plus supports the `--inspect` flag natively (otherwise, you'd need to somehow invoke it with a `node` command or pass it in from the top level or something). The quirk is that you have to refer to the `node_modules` version of jasmine, but I was unable to get this to work with the normal compile --> js version (was receiving "jasmine is not a module" errors).

I did remove the unecessary compile step to simplify it down to its most necessary parts.